pkg/deploy/lattice/targets_manager.go

```diff
@@ -61,39 +62,33 @@ func (s *defaultTargetsManager) registerTargets(
 	modelTargets *model.Targets,
 	modelTg *model.TargetGroup,
 ) error {
-	var latticeTargets []*vpclattice.Target
-	for _, modelTarget := range modelTargets.Spec.TargetList {
-		port := modelTarget.Port
-		targetIP := modelTarget.TargetIP
-		t := vpclattice.Target{
-			Id:   &targetIP,
-			Port: &port,
-		}
-		latticeTargets = append(latticeTargets, &t)
-	}
-
 	// No targets to register
-	if len(latticeTargets) == 0 {
+	if len(modelTargets.Spec.TargetList) == 0 {
 		return nil
 	}
```

Comment:
redundant condition, loop below holds invariant

Reply:
Return earlier to reduce cognitive load?

Reply:
Less code reduces cognitive load, de-duplicating code also reduces cognitive load. Redundant checks and validations create fuzz around important code.

Reply:
Removed